lib/wp2middleman/frontmatter.rb

```diff
@@ -10,7 +10,7 @@ def initialize(post, include_fields: [])
     def post_data
       data = {
         'title' => post.title,
-        'date' => post.date_published,
+        'date' => post.date_time_published,
```

Comment:
Hmm, looking at the definitions of `#valid?`, and the others don't think it'll make any difference:

```ruby
    def valid?
      !(post_date.nil? || title.nil? || date_published.nil? || content.nil?)
    end

    # ...

    def date_published
      Date.parse(post_date).to_s
    end

    def date_time_published
      Time.parse(post_date).strftime("%F %T")
    end
```

Neither `#date_published` nor `#date_time_published` can ever be `#nil?`--they always return strings--so that clause of `#valid?` will always be false. 

 (If there's no date field, `#post_date` will be `nil`, and the boolean expression will short circuit without even trying to evaluate the rest.  And if `#post_date` returns an invalid string, `#date_published` will throw an error and not return anything.)